packages/block-library/src/post-template/edit.js

```diff
@@ -95,11 +96,23 @@ export default function PostTemplateEdit( {
 		} = {},
 		queryContext = [ { page: 1 } ],
 		templateSlug,
-		displayLayout: { type: layoutType = 'flex', columns = 1 } = {},
+		displayLayout: { type: layoutType = 'grid', columnWidth = 12 } = {},
 		previewPostType,
 	},
 	__unstableLayoutClassNames,
 } ) {
+	const updatedLayoutType =
+		layoutType === 'list' || layoutType === 'default' ? 'default' : 'grid';
+
+	useEffect( () => {
```

Comment:
I don't love this but don't think there's a way around it unless we move the list/grid controls into Post Template instead of keeping them in Query.

Reply:
It might be worth looking into exploring moving them down 🤔